app/views/tasks/index.html.erb

```diff
@@ -0,0 +1,44 @@
+<h1> Welcome to your to-do list! Your tasks are: </h1>
+
+<div class = "tasklist">
+  <% @task_list.each do |task| %>
+  <% if @task_list == nil %>
+  <%= puts "You have no tasks to list." %>
+  <% end %>
+  <% end %>
+
+  <table>
+    <thead>
+      <tr>
+        <th>Title</th>
+        <th>Description</th>
+        <th>Status</th>
+        <th>Due Date</th>
+        <th>Completed on</th>
+        <th>Task Owner</th>
+        <th>Action</th>
+      </tr>
+    </thead>
+    <tbody>
+      <% @task_list.each do |task| %>
+      <tr>
+        <td><%= link_to task.task_name, "/tasks/#{task.id}" %></td>
```

Comment:
Could you clarify this for me? I'm not sure I know exactly what you mean. Thank you!